records_mover/records/delimited/types.py

```diff
@@ -71,3 +58,44 @@
                    "timeonlyformat",
                    "datetimeformattz",
                    "datetimeformat"]
+
+HINT_NAMES: List[HintName] = list(get_args(HintName))  # type: ignore
+
+# TODO: Retire this
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker